Sources/UILocalNotification+Sweet.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+public extension UILocalNotification {
+    static let idKey = "idKey"
+    static let fireDateKey = "fireDateKey"
+
+    public static func create(_ id: String, fireDate: Date, soundName: String? = nil, message: String, actionTitle: String? = nil) {
+        let notification = UILocalNotification()
+        notification.soundName = soundName
+
+        notification.fireDate = fireDate
+        notification.timeZone = TimeZone.current
+        notification.alertBody = message
+        notification.alertAction = actionTitle
```

Comment:
I was thinking something like this, so the two cases are separated. A case where there is an action, and the one where there's none.
```
if let _actionTitle = actionTitle {
 notification.alertAction = _actionTitle
} else {
notification.hasAction = false
}
```